solutions/CF Kayaking.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: cf-kayaking
+source: CF
+title: Kayaking
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/54708)
+
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
+	int N;
+	cin >> N;
+	N *= 2;
+	vector<int> W(N);
+	for (int& x : W) cin >> x;

```

Comment:
```cpp 
for(int i=0; i<N; i++){
cin >> W[i];
}
```

Reply:
I think there is no need, everyone has personal preference writing code. Anyways, @darren-yao  decides.